### PR TITLE
SUBMARINE-708. Add mailing list subscription link

### PR DESCRIPTION
**Pull request by aeioulisa**
### What is this PR for?
Add mailing list subscription link in Readme

### What type of PR is it?
[Documentation]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-708

### Screenshots (if appropriate)
image

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: docs/community/README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Based on your track-record as a contributor, Per Apache code, PMCs vote on commi
 Note that becoming a committer is not just about submitting some patches; it‘s also about helping out on the development and user [Slack User](https://the-asf.slack.com/submarine-user/), helping with documentation and the issues.
 
 See [Become a Committer.md](become-a-committer.md) for steps of becoming a committer and more details.
-
+Mailing list
```
Comment (cxorm) [inline docs/community/README.md:60]: IMHO it is a  redundant line, 
feel free to let me know why we need this line if I miss something.

**Review by cxorm — COMMENTED** [review]
Thanks @aeioulisa for the work. 
Overall LGTM, one question inlined.

**Review by cxorm — APPROVED** [review]
Thank you @aeioulisa for the fix.
I'm +1 for it.

**Comment by cxorm** [comment]
Hi @pingsutw,
Could you be so kind as to take a look on this PR, thanks.

**Review by pingsutw — APPROVED** [review]
+1

File: docs/community/README.md [hunk 2]
```diff
@@ -66,6 +66,17 @@ See [How to commit](HowToCommit.md) for helper doc for Submarine committers.
 
 Communication within the Submarine community abides by [Apache’s Code of Conduct](https://www.apache.org/foundation/policies/conduct.html).
 
+## Mailing list
```
Comment (lowc1012) [inline docs/community/README.md:69]: Mailing lists

File: docs/community/README.md [hunk 3]
```diff
@@ -66,6 +66,17 @@ See [How to commit](HowToCommit.md) for helper doc for Submarine committers.
...
 
+## Mailing lists
+
+Get help using Apache Submarine or contribute to the project on our mailing lists:
+
++ [Users](https://lists.apache.org/list.html?users@submarine.apache.org) : [subscribe](users-subscribe@submarine.apache.org), [unsubscribe](users-unsubscribe@submarine.apache.org), [archives](https://lists.apache.org/list.html?users@submarine.apache.org)
+for usage questions, help, and announcements.
++ [Dev](https://lists.apache.org/list.html?dev@submarine.apache.org) : [subscribe](dev-subscribe@submarine.apache.org), [unsubscribe](dev-unsubscribe@submarine.apache.org), [archives](https://lists.apache.org/list.html?dev@submarine.apache.org)
+for people wanting to contribute to the project.
++ [Commits](https://lists.apache.org/list.html?commits@submarine.apache.org) : [subscribe](commits-subscribe@submarine.apache.org), [unsubscribe](commits-unsubscribe@submarine.apache.org), [archives](https://lists.apache.org/list.html?commits@submarine.apache.org)
```
Comment (lowc1012) [inline docs/community/README.md:77]: The links for subscribing and unsubscribing should be "mailto:xxxxx@submarine.apache.org"

**Review by lowc1012 — CHANGES_REQUESTED** [review]

**Review by lowc1012 — APPROVED** [review]
LGTM. @aeioulisa  Thanks!

**Review by xunliu — APPROVED** [review]
@aeioulisa Thank you for contributions.
Will merge if no more comments.

**Pull request closed without merging**